hypervisor/src/kvm/mod.rs

```diff
@@ -1030,7 +1041,8 @@ impl cpu::Vcpu for KvmVcpu {
                     Ok(cpu::VmExit::MmioWrite(addr, data))
                 }
                 VcpuExit::Hyperv => Ok(cpu::VmExit::Hyperv),
-
+                #[cfg(feature = "tdx")]
+                VcpuExit::Unsupported(KVM_EXIT_TDX) => Ok(cpu::VmExit::Tdx),
```

Comment:
For now, since you're saying everything is unsupported you can handle it inside here rather than propagating the exit up to the VMM loop.

Reply:
Yes we can also take this approach but I've implemented your proposal about `TdxExitDetails` which is quite clean and will let us concentrate on the actual implementations of the different VMCALLs once we get to it.